broker-cli/commands/order/index.js

```diff
@@ -61,17 +61,17 @@ module.exports = (program) => {
       }
     })
     .command(`order ${SUPPORTED_COMMANDS.SUMMARY}`, 'View your orders')
+    .option('--market <marketName>', MARKET_NAME_HELP_STRING, validations.isMarketName, null, true)
```

Comment:
just an FYI. None of these validations or defaults are used (they dont affect the sub commands because of the way we've rigged it). We need to actually run the marketname validation in the `.action` handler for subcommands

Reply:
hmm. Do you think we should just remove them then? This PR adds a bunch of them back in, but if they aren't used it's a bit strange.